ca/config.go

```diff
@@ -472,18 +480,12 @@ func RenewTLSConfig(ctx context.Context, s *SecurityConfig, connBroker *connecti
 					return
 				}
 			} else {
-				// If we have an expired certificate, we let's stick with the starting default in
-				// the hope that this is a temporary clock skew.
+				// If we have an expired certificate, try to renew immediately: the hope that this is a temporary clock skew, or
```

Comment:
I originally set it to renew every few seconds because it could take a little while for the cluster to start up in the `ForceNewCluster` case (since we have to wait for the cluster data to load).

But it renews based on exponential backoff, so given the current configuration, it should be:

1. `0s`
1.  `35s`
1.  `1m5s`
1.  `2m5s`
1.  `4m5s`

and then maxes out at 5 minutes (so it renews every 5 minutes).  Although looking at the actual exponential backoff code, the actual time seems to be chosen from a uniform distribution between 0 and those times, so could be every few seconds if unlucky.

We could max it out at 1 minute instead.